pom.xml

```diff
@@ -588,6 +588,11 @@
         <artifactId>adam-codegen-spark2_${scala.version.prefix}</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.thoughtworks.paranamer</groupId>
```

Comment:
What are you using this for?

Reply:
More classpath woes
https://issues.apache.org/jira/browse/SPARK-26146
https://issues.apache.org/jira/browse/SPARK-26583

Reply:
Ahh I see

Reply:
It's been very frustrating dealing with the Spark folks, they keep going fast and breaking things without cleaning up any of the dependency mess left in the wake